### DISCUSSION TITLE
Search Filters Video

### DISCUSSION THREAD

**Discussion by blublu80** [libre-tube/LibreTube#6432] [Ideas]
would be useful in the search filters to search for videos like there is on youtube would be EXCELLENT

**Comment by san616mofo** [comment]
+1
Important feature: search filters, like those available in the official YouTube app or website. Currently, when I search for something, I often get results that are several years old. This makes it difficult to find relevant or recent content.

For example, I often want to filter videos by upload date, such as "last month" or "this week." Not having this option is actually the main reason I uninstalled the app. I need access to up-to-date content — not just anything vaguely related from years ago.

Please consider adding this feature. I believe many users would benefit from more control over their search results.

**Comment by adultbrian** [comment]
I know this isn't perfect but a decent work around I have been using is to type before:Year-Mo-Da or after:Year-Mo-Da works great
For example today is December 10, 2025 so if i wanted videos for the last day I would type after:2025-12-09 then my search

**Comment by divyanshsethi24-art** [comment]
What about switching to another extractor if NewPipe is currently not supporting `Sort Search Filter`

Other NewPipe forks already support Sort search filter can `LibreTube` swtich to their extractor? 

https://github.com/bravepipeproject/BravePipe
https://github.com/InfinityLoop1308/PipePipe
https://github.com/InfinityLoop1308/PipePipe-X
